### ISSUE TITLE
Windows binary is x86_64

### ISSUE THREAD

**Issue by carlocorradini** [vscode-shellcheck/shellcheck-binaries#8]
```sh
$ file shellcheck.exe
shellcheck.exe: PE32+ executable (console) x86-64 (stripped to external PDB), for MS Windows
```

We need to change the archive name back to `x86_64`

**Comment by carlocorradini** [comment]
This update is easy.

From:
```sh
download_and_archive "windows" "x86"
```

To:
```sh
download_and_archive "windows" "x86_64"
```

**Comment by carlocorradini** [comment]
Fixed in PR #9
